Traceback (most recent call last):
  File "/home/robertdefilippi/modo-sarima-model/pythonScript.py", line 127, in <module>
    'lng': lng_values
NameError: name 'lng_values' is not defined
Traceback (most recent call last):
  File "/home/robertdefilippi/modo-sarima-model/pythonScript.py", line 127, in <module>
    'lng': lng_values
NameError: name 'lng_values' is not defined
